inductiva/tasks/task.py

```diff
@@ -156,7 +156,8 @@ def wait(self, polling_period: int = 5) -> models.TaskStatusCode:
                     logging.info("Task failed.")
                     logging.info("Download the task output and check the "
```

Comment:
Does this log need to change as well, since those files are automatically downloaded?